chromium_src/extensions/common/mojom/api_permission_id.mojom

```diff
@@ -1,12 +0,0 @@
-// Copyright (c) 2021 The Brave Authors. All rights reserved.
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this file,
-// You can obtain one at http://mozilla.org/MPL/2.0/.
-
-module extensions.mojom;
-
-[BraveExtend]
-enum APIPermissionID {
```

Comment:
from https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/mojom/api_permission_id.mojom;l=10?q=api_permission_id.mojom&ss=chromium

it seems we cannot just delete these entries